package.json

```diff
@@ -12,9 +12,15 @@
     "@angular/platform-browser": "~11.2.4",
     "@angular/platform-browser-dynamic": "~11.2.4",
     "@angular/router": "~11.2.4",
+    "@fortawesome/angular-fontawesome": "^0.8.2",
+    "@fortawesome/fontawesome-svg-core": "^1.2.34",
+    "@fortawesome/free-regular-svg-icons": "^5.15.3",
+    "@fortawesome/free-solid-svg-icons": "^5.15.3",
     "@ng-bootstrap/ng-bootstrap": "^9.0.2",
-    "@solgenomics/brapijs": "git+ssh://git@github.com:solgenomics/BrAPI-js.git#semver:^1.0.1",
+    "@ng-select/ng-select": "6.1.0",
```

Comment:
Nice, this one seems like it's much more supported than the alternatives, I don't know why I did't find it last time I wanted something similar..